server/graphql/v1/inputTypes.js

```diff
@@ -67,6 +67,18 @@ export const PaymentMethodDataVirtualCardInputType = new GraphQLInputObjectType(
   }),
 });
 
+export const CustomFieldsInputType = new GraphQLInputObjectType({
+  name: 'CustomFieldsInputType',
+  description: 'Input for custom fields for order',
+  fields: () => ({
+    type: { type: GraphQLString },
```

Comment:
This should be an enum, so we know the allowed values.

Reply:
Enum of?

Reply:
All possible values for `type`.

Reply:
okay.